docs/docs/adding-react-components.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Adding React Components
+---
+
+This guide covers how to add React components to your Gatsby site.
+
+## React components
+
+React components are prebuilt elements or groups of elements that can be used to prevent repeating yourself in your code.
+
+Components also offer the ability to be customized using "props" (properties). Props can be of any Javascript type, such as boolean, string, object, array or almost anything you can think of.
+
+For example, you could use a component for Buttons on your site. This is because they would likely be used multiple times across pages and would need different labels each time.
```

Comment:
```suggestion
For example, you could use a component for Buttons on your site. This would enable them to be used multiple times across pages with different labels or actions each time.
```